<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HearthMod Beta</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../.wine/drive_c/games/HS3/Hearthstone_Data/wait.cur</normaloff>../../../.wine/drive_c/games/HS3/Hearthstone_Data/wait.cur</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>731</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Logs</string>
    </property>
    <widget class="QListWidget" name="listLogs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>711</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="lobby">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>731</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Lobby</string>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>341</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Login</string>
     </property>
     <widget class="QLineEdit" name="editUsername">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>231</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>67</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="editPassword">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>231</width>
        <height>25</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>100</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Player</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="player1">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="player2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>30</y>
       <width>351</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Game</string>
     </property>
     <widget class="QCheckBox" name="checkDeck">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>92</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Deck</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushPlay">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Play!</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkAssets">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>92</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Assets</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
